packaging/markers.py

```diff
@@ -84,7 +84,8 @@ class Value(Node):
 ALIASES = {
     'python_implementation': 'platform_python_implementation'
 }
-VARIABLE.setParseAction(lambda s, l, t: Variable(ALIASES.get(t[0], t[0].replace('.', '_'))))
+VARIABLE.setParseAction(lambda s, l, t:
+                        Variable(ALIASES.get(t[0], t[0].replace('.', '_'))))
```

Comment:
I full confess this is hideous. But it's overwritten by the next commit, and it keeps `tox` passing on every commit (once squashed), so hey.